### PR TITLE
Fix issues with <img> element's loading property

### PR DESCRIPTION
**Pull request by a2sheppy**
Resolve issue #2924 fully by setting the `experimental` flag to `false` for this property.

Also realized the property was missing in `HTMLImageElement`, so I added that.

### CODE REVIEW

File: api/HTMLImageElement.json [hunk 1]
```diff
@@ -669,6 +669,56 @@
           }
         }
       },
+      "loading": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": "76"
```
Comment (jpmedley) [inline api/HTMLImageElement.json:676]: Actually this landed in 77: https://storage.googleapis.com/chromium-find-releases-static/0fc.html#0fc50ca2d3c81f74f73aae8aab6079a516d9b8fb

File: api/HTMLImageElement.json [hunk 2]
```diff
@@ -669,6 +669,56 @@
...
+            },
+            "edge": {
+              "version_added": "79"
+            },
+            "firefox": {
+              "version_added": "75"
+            },
+            "firefox_android": {
+              "version_added": false,
+              "notes": "See <a href='https://bugzil.la/1542784'>bug 1542784</a>"
```
Comment (Elchi3) [inline api/HTMLImageElement.json:689]: This links to a fixed bug. I think we should not add it. I don't see anything useful in there for Fx Android.

```suggestion
              "version_added": false
```

**Comment by ddbeck** [comment]
@a2sheppy hi Sheppy, do you plan to come back to this PR?

**Comment by Elchi3** [comment]
This was actually done in https://github.com/mdn/browser-compat-data/pull/5862

**Pull request closed without merging**